css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,36 @@
+.container {
+  font-family: "DM Sans", sans-serif;
+  width: 1200px;
```

Comment:
It might be worth considering `max-width` over `width` to provide more flexibility and responsiveness to your layout.

The width property sets the width of an element, and the element will always have that fixed width, regardless of the screen size or device used to view it. This can cause issues with responsiveness, as the element may become too small or too large on different screen sizes.

On the other hand, the max-width property sets the maximum width of an element, which allows the element to be responsive and adapt to different screen sizes. When the screen size is smaller than the max-width value, the element will shrink to fit the screen, but when the screen size is larger than the max-width value, the element will stop growing and stay at that maximum width.

Reply:
thank you! will keep that in mind for the future